CONTRIBUTING.md

```diff
@@ -57,23 +61,50 @@ Initiating the next minor release after releasing `X.Y.0`:
 
 Other notes:
 
-* Patch releases are _not_ merged to `dev`
+* Patch releases are _only_ merged to `dev` if they are part of the most recent release line (currently 3.1, which will shift to 3.2 once 3.2.0 is released).
```

Comment:
This way `src/oas.md` will be the overall latest published version, which is nice.

However, the `versions` folder will be incomplete and only contain a subset of the files available in `main`.

This feels wrong, but I can't put my finger on what exactly bothers me.

Reply:
@ralfhandl that feels wrong because it's not what is happening. 🙂  I'll clarify.